conboarding.md

```diff
@@ -89,6 +89,16 @@ How much does it cost to run things?
       ```
       With this ssh config in place, `ssh typhon.dartmouth.edu` would
       jump you over through `smaug`.
+   1. If you need to use MacOS, ask Yarik to add your key to the Con Mac.
+      Then you can add to your ssh config in `smaug` and login from there.
+      ```
+      Host conmac
+        Hostname datalads-imac2.kiewit.dartmouth.edu
```

Comment:
I think it should be safe to disclose since private network, but getting close to "secrets".